### DISCUSSION TITLE
Auto hide scrollbar on tabs panel collapse

### DISCUSSION THREAD

**Discussion by zenwav** [Floorp-Projects/Floorp#940] [Feature Request]
Currently, the scrollbar stays visible even when the tabs panel is collapsed, leading to unnecessary visual clutter. It would enhance the user experience if the scrollbar could auto-hide after the panel is collapsed, similar to the functionality in Microsoft Edge.

**Comment by TheCleric** [comment]
Came here to suggest the same. When collapsed, the scrollbar takes up way too much horizontal space (and is not visually appealing in my opinion).

**Comment by johnsonjamees** [comment]
I would also like to see this implemented.

**Comment by creeper-0910** [comment]
I will check.

**Comment by zenwav** [comment]
Any updates yet?

**Comment by creeper-0910** [comment]
This will be fixed in the next version.

**Comment by creeper-0910** [comment]
Sorry, please wait for 11.17.9.
It doesn't seem to be working correctly.
